app/apps/server/bridges/http.ts

```diff
@@ -6,6 +6,8 @@ import { IHttpBridgeRequestInfo } from '@rocket.chat/apps-engine/server/bridges'
 import { AppServerOrchestrator } from '../orchestrator';
 import { getUnsafeAgent } from '../../../../server/lib/getUnsafeAgent';
 
+const isGetOrHead = (method: string): boolean => ['GET', 'HEAD'].includes(method.toUpperCase());
```

Comment:
it hurts me deep down in my heart having to upper case `method` every time 🙈

Reply:
Calling a method every time is bad but creating an array every time doesn't raise any flags? 😛

Reply:
yep.. I agree hence I didn't ask for changes.. I was just expressing my feeling :P